config/initializers/warden.rb

```diff
@@ -8,8 +8,9 @@
 
 require 'warden/strategies/basic_strategy'
 require 'warden/strategies/session_strategy'
+require "warden/strategies/doorkeeper"
 
 Rails.application.config.middleware.insert_after(ActionDispatch::Flash, Warden::Manager) do |manager|
-  manager.default_strategies :basic, :session
+  manager.default_strategies :doorkeeper, :basic, :session
```

Comment:
We want doorkeeper to only work on api urls. 

I can look into ways of blocking it otherwise.